src/form/ColormapAllSteps.tsx

```diff
@@ -115,9 +115,36 @@ export const ColormapAllSteps: React.FC<Props> = (props) => {
               <div
                 style={{position: "relative"}}
               >
-                <button 
-                  onClick={()=> {if (visiblepickerIndex === ind) {setVisiblepickerIndex(null)} else {setVisiblepickerIndex(ind)} }}
-                  type="button"
+                {/* 
+                  following div should act as a button.
+                  Button was turned into div, because it needs nested divs for the colorpicker.
+                  These divs need to be nested because clicking on them need to keep this div in focus 
+                  The div now namely uses a onblur to close the colorpicker (as replacement of the overlay with onclick previously used)
+                */}
+                <div 
+                  // this tabIndex is needed to make sure the div (that should actually be a button, but was made div for above reasons) does receive onfocus and onblur event
+                  tabIndex={0}
+                  onBlur={(event)=> {
+                    console.log('event', event)
+                    setVisiblepickerIndex(null);
+                  }}
+                  // Todo:
+                  // I added the keypress so user can also tab enter open this colorpicker.
+                  // anyway now the user might expect that excape closes it.
+                  // Not only does this not work (see next line, does not distinct between keypresses), 
+                  // also the event listener on <ModalBackground/> closes the entire modal on escape
+                  onKeyPress={()=>setVisiblepickerIndex(ind)}
```

Comment:
I think this added feature of using enter to open the color picker is not necessary. You still have to use your mouse to pick a color anyway. And you can use ENTER to open the color picker, but cannot close it using either 'Enter' nor 'ESC' so in the end, user is left to use the mouse anyway to use the color picker. I also noticed that the current version on staging works slightly better as you can use ENTER to open or close the color picker modal (not with ESC though).

Reply:
commented it out for now